scout/server/blueprints/institutes/controllers.py

```diff
@@ -598,13 +603,37 @@ def get_and_set_cases_by_status(
             case_groups[status].append(case_obj)
             nr_cases_showall_statuses += 1
 
+    def get_specific_query(request):
+        """Check if only (non-)specific query terms were used in query.
+        If so we assume this is a default query, and dim all cases
+        that match the "show_all_cases_status", highlighting the
+        (max limit number) other cases that were returned.
+        """
+        cases_query = store.cases(
```

Comment:
Why are you running this query again, you should already have all cases returned by the exact same query, see [https://github.com/Clinical-Genomics/scout/blob/be77892e1e2ce15ac095affeb484a6d917b1fabe/scout/server/blueprints/institutes/controllers.py#L638]